wrappers/swift/Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "1DSSwiftWrapper",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "SwiftWrapper",
```

Comment:
Can we have this so that we can `import OneDS` or `import OneDSSwift` in our code, please?